### ISSUE TITLE
Trinity login node: disable selinux

### ISSUE THREAD

**Issue by hansthen** [clustervision/trinity#282]
Needs to be discussed in more detail as the login node is one of the first points of attack from inside.

**Comment by themisa** [comment]
Addressed in aa08d3916c8d

**Comment by themisa** [comment]
**Version:** r6 aa08d39
**Environment:** tmaster


```
[jane@login ~]$ sestatus
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   permissive
Mode from config file:          permissive
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Max kernel policy version:      28
[jane@login ~]$ ^C
[jane@login ~]$ sestatus
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   permissive
Mode from config file:          permissive
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Max kernel policy version:      28
```
